app/assets/stylesheets/application.scss

```diff
@@ -54,6 +54,14 @@
   }
 
 //panel elements
+
+  .fullWidth {
```

Comment:
Line should be indented 0 spaces, but was indented 2 spaces<br>Selector `fullWidth` should be written in lowercase with hyphens